src/Java.Interop.Export/Tests/Java.Interop/ExportTest.cs

```diff
@@ -11,11 +11,11 @@ namespace Java.InteropTests
 	[JniTypeSignature ("com/xamarin/interop/export/ExportType")]
 	public class ExportTest : JavaObject
 	{
-		static void __RegisterNativeMembers (JniType type, string members)
+		[JniAddNativeMethodRegistrationAttribute]
+		static void RegisterNativeMembers (JniNativeMethodRegistrationArguments args)
 		{
-			var methods = JniEnvironment.Runtime.MarshalMemberBuilder
-				.GetExportedMemberRegistrations (typeof (ExportTest));
-			type.RegisterNativeMethods (methods.ToArray ());
+			args.AddRange (JniEnvironment.Runtime.MarshalMemberBuilder
+			               .GetExportedMemberRegistrations (typeof (ExportTest)));
```

Comment:
Your editor is weird, and is inserting spaces. This should be two tab stops before the `.GetExportedMemberRegistrations()`.